lib/browser.ts

```diff
@@ -216,19 +216,13 @@ export class ProtractorBrowser extends AbstractExtendedWebDriver {
    *
    * Initialized to `false` by the runner.
    *
-   * This property is deprecated - please use waitForAngularEnabled instead.
+   * ignoreSynchornization is deprecated.
```

Comment:
If we're making this actually, totally private then we can remove this jsdoc.

Reply:
Done.